### DISCUSSION TITLE
Where do I find release info?

### DISCUSSION THREAD

**Discussion by sambauers** [atlassian/pragmatic-drag-and-drop#69] [General]
Hi, your library is great. Just the right level of abstraction and modularity.

I just got notice of your latest release, but it is very difficult to see what has changed through GitHub. Would you consider adding a simple change log with links to commits that are part of each release? Or just a link to the compare view between versions on Github?

**Comment by sambauers** [comment]
Just found the change log buried in packages…

https://github.com/atlassian/pragmatic-drag-and-drop/blob/main/packages/core/CHANGELOG.md

**Comment by eddiemonge** [comment]
Hoping to bring this back up because it took searching through discussions to find something that should be more prominent
